src/solvers/dgsem_tree/dg_1d_parabolic.jl

```diff
@@ -433,6 +471,39 @@ function calc_gradient_interface_flux!(surface_flux_values,
     end
 end
 
+function calc_gradient_interface_flux!(surface_flux_values,
+                                       mesh::TreeMesh{1}, equations_parabolic, dg::DG,
+                                       parabolic_scheme::ViscousFormulationLocalDG,
+                                       cache, cache_parabolic)
+    @unpack neighbor_ids, orientations = cache_parabolic.interfaces
+
+    @threaded for interface in eachinterface(dg, cache_parabolic)
+        # Get neighboring elements
+        left_id = neighbor_ids[1, interface]
+        right_id = neighbor_ids[2, interface]
+
+        # Determine interface direction with respect to elements:
+        # orientation = 1: left -> 2, right -> 1
+        left_direction = 2 * orientations[interface]
+        right_direction = 2 * orientations[interface] - 1
+
+        # Call pointwise Riemann solver
+        u_ll, u_rr = get_surface_node_vars(cache_parabolic.interfaces.u,
+                                           equations_parabolic, dg, interface)
+        flux = 0.5f0 * (u_ll + u_rr)
+        flux_jump = 0.5f0 * (u_rr - u_ll)
+
+        # Copy flux to left and right element storage
+        for v in eachvariable(equations_parabolic)
+            # Here, the flux is {{f}} + beta * [[f]], where beta is the LDG "switch", 
+            # which we set to -1 on the left and +1 on the right in 1D. The sign of the 
+            # jump term should be opposite that of the sign used in the divergence flux. 
+            surface_flux_values[v, left_direction, left_id] = flux[v] - flux_jump[v]
+            surface_flux_values[v, right_direction, right_id] = flux[v] + flux_jump[v]
```

Comment:
```suggestion
            # Here, the flux is {{f}} + beta * [[f]], where beta is the LDG "switch", 
            # which we set to -1 on the left and +1 on the right in 1D. The sign of the 
            # jump term should be opposite that of the sign used in the divergence flux. 
            # This is equivalent to
            # - u_ll for the left_direction, left_id
            # - u_rr for the right_direction, right_id
            surface_flux_values[v, left_direction, left_id] = flux[v] - flux_jump[v]
            surface_flux_values[v, right_direction, right_id] = flux[v] + flux_jump[v]
```

Reply:
Then, we would need to change this to

```suggestion
            # Here, the flux is {{f}} + beta * [[f]], where beta is the LDG "switch", 
            # which we set to -1 on the left and +1 on the right in 1D. The sign of the 
            # jump term should be opposite that of the sign used in the divergence flux. 
            # This is equivalent to u_ll.
            val = flux[v] - flux_jump[v]
            surface_flux_values[v, left_direction, left_id] = val
            surface_flux_values[v, right_direction, right_id] = val
```